setup.py

```diff
@@ -38,6 +38,7 @@
       extras_require=dict(
           docs=['sphinx', 'shinx_rtd_theme', 'numpydoc'],
           demo=['matplotlib'],
+          sdml=['skggm']
```

Comment:
Can we specify a commit hash here? Or maybe since their latest release is 0.2.8, we could specify `skggm>=0.2.9`.

Reply:
I tried indeed but didn't manage to make it work. But good idea, skggm>=0.2.9 seems is better than nothing here